src/lib/assets/index.ts

```diff
@@ -28,6 +29,68 @@ import { watcher, moduleSecret } from "./pods";
 
 import { clusterRoleBinding, serviceAccount, storeRole, storeRoleBinding } from "./rbac";
 import { createDirectoryIfNotExists } from "../filesystemService";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Was this added intentionally or as a stopgap while working on the complexity warnings?

Reply:
This was intentional.

This function has the responsibility to take **any** (legit!) JS object -- regardless of type -- and pass it through to a yaml serializer for stringifying; doesn't do any validations / property accesses and so shouldn't really have anything to say about the types passed in, IMO.